Filtered_kernel/include/CGAL/Filtered_kernel/internal/Static_filters/Power_side_of_oriented_power_circle_2.h

```diff
@@ -0,0 +1,335 @@
+      init_double(tx, ty, twt, (FT*)(0));
+      if( fit_in_double(p.x(), px) && fit_in_double(p.y(), py)       &&
+          fit_in_double(p.weight(), pwt) &&
+          fit_in_double(q.x(), qx) && fit_in_double(q.y(), qy)       &&
+          fit_in_double(q.weight(), qwt) &&
+          fit_in_double(r.x(), rx) && fit_in_double(r.y(), ry)       &&
+          fit_in_double(r.weight(), rwt) &&
+          fit_in_double(t.x(), tx) && fit_in_double(t.y(), ty)       &&
+          fit_in_double(t.weight(), twt)
+          )
+        {
+          CGAL_BRANCH_PROFILER_BRANCH_1(tmp);
+          double dpx;
+          dpx = (px - tx);
+          double dpy;
+          dpy = (py - ty);
+          double dpz;
+          dpz = (((square( dpx ) + square( dpy )) - pwt) + twt);
+          double dqx;
+          dqx = (qx - tx);
+          double dqy;
+          dqy = (qy - ty);
+          double dqz;
+          dqz = (((square( dqx ) + square( dqy )) - qwt) + twt);
+          double drx;
+          drx = (rx - tx);
+          double dry;
+          dry = (ry - ty);
+          double drz;
+          drz = (((square( drx ) + square( dry )) - rwt) + twt);
+          result_type int_tmp_result;
+          double RT_tmp_result;
+          double eps;
+          RT_tmp_result = CGAL::determinant( dpx, dpy, dpz, dqx, dqy, dqz, drx, dry, drz );
+          double max1;
+          double max3 = CGAL::abs(dpy);
+          if( (max3 < CGAL::abs(dqy)) )
+            {
+              max3 = CGAL::abs(dqy);
+            }
+          if( (max3 < CGAL::abs(dry)) )
+            {
+              max3 = CGAL::abs(dry);
+            }
+          max1 = max3;
+          double max2 = CGAL::abs(dpx);
+          if( (max2 < CGAL::abs(dqx)) )
+            {
+              max2 = CGAL::abs(dqx);
+            }
+          if( (max2 < CGAL::abs(drx)) )
+            {
+              max2 = CGAL::abs(drx);
+            }
+          if( (max1 < max2) )
+            {
+              max1 = max2;
+            }
+          double max4 = CGAL::abs(pwt);
+          if( (max4 < CGAL::abs(qwt)) )
+            {
+              max4 = CGAL::abs(qwt);
+            }
+          if( (max4 < CGAL::abs(rwt)) )
+            {
+              max4 = CGAL::abs(rwt);
+            }
+          if( (max4 < CGAL::abs(twt)) )
+            {
+              max4 = CGAL::abs(twt);
+            }
+          double lower_bound_1;
+          double upper_bound_1;
+          lower_bound_1 = max3;
+          upper_bound_1 = max3;
+          if( (max1 < lower_bound_1) )
+            {
+              lower_bound_1 = max1;
+            }
+          else
+            {
+              if( (max1 > upper_bound_1) )
+                {
+                  upper_bound_1 = max1;
+                }
+            }
+          if( (max2 < lower_bound_1) )
+            {
+              lower_bound_1 = max2;
+            }
+          if( ((lower_bound_1 < 2.99168207048872973507e-74) || (max4 < 8.95016161088373414772e-148)) )
+            {
```

Comment:
Wouldn't it be better to have a const or a define for this to tell what the values mean / is the idea behind it?